core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -77,6 +80,9 @@ private[spark] object ClosureCleaner extends Logging {
     Nil
   }
 
+  /**
+   * Return a list of classes that represent closures enclosed in the given closure object.
+   */
   private def getInnerClasses(obj: AnyRef): List[Class[_]] = {
```

Comment:
Can we do validations here that `obj` is a closure type (for instance doing an instanceOf check with an exception)? It would add a bit more type safety and clarity to the API.